features/gov_uk.feature

```diff
@@ -12,3 +12,16 @@ Feature: Core GOV.UK behaviour
     And I force a varnish cache miss
     When I visit "/"
     Then the logo should link to the homepage
+
+  @normal
+  Scenario: entirely upper case slugs redirect to lowercase
+    Given I visit "https://www.gov.uk/GOVERNMENT/PUBLICATIONS" without following redirects
```

Comment:
I think these tests should both drop the hostname and start with a leading slash so that they test the right thing in integration and staging.